h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -1161,14 +1178,31 @@ public GLMOutput(DataInfo dinfo, String[] column_names, String[] column_types, S
 
     public GLMOutput(DataInfo dinfo, String[] column_names, String[] column_types, String[][] domains, String[] coefficient_names, boolean binomial, double[] beta) {
       this(dinfo,column_names,column_types, domains,coefficient_names,binomial);
-      assert !ArrayUtils.hasNaNsOrInfs(beta);
+      assert !ArrayUtils.hasNaNsOrInfs(beta): "Coefficients contain NA or Infs.";
       _global_beta=beta;
       _submodels = new Submodel[]{new Submodel(0,beta,-1,Double.NaN,Double.NaN)};
     }
 
+    public GLMOutput(DataInfo dinfo, String[] column_names, String[] column_types, String[][] domains, 
+                     String[] coefficient_names, double[] beta, boolean isMultinomial) {
+      super(dinfo._weights, dinfo._offset, dinfo._fold);
```

Comment:
do we need to copy past this code? I think there is a way how to extend the current
`    public GLMOutput(DataInfo dinfo, String[] column_names, String[] column_types, String[][] domains, String[] coefficient_names, boolean binomial) {
`
contructor and just call this() here

Reply:
Deleted.